docs/integrations/containers-orchestration/opentelemetry/vmware-opentelemetry.md

```diff
@@ -0,0 +1,254 @@
+  values={[
+    {label: 'Linux', value: 'Linux'},
+    {label: 'Windows', value: 'Windows'},
+    {label: 'macOS', value: 'macOS'},
+    {label: 'Chef', value: 'Chef'},
+    {label: 'Ansible', value: 'Ansible'},
+    {label: 'Puppet', value: 'Puppet'},
+  ]}>
+
+<TabItem value="Linux">
+
+1. Copy the yaml file to `/etc/otelcol-sumo/conf.d/` folder in the VMWare instance which has access to vCenter or the vCenter itself.
+2. Restart the collector using:
+  ```sh
+  sudo systemctl restart otelcol-sumo
+  ```
+
+</TabItem>
+<TabItem value="Windows">
+
+1. Copy the yaml file to `C:\ProgramData\Sumo Logic\OpenTelemetry Collector\config\conf.d` folder in the VMWare instance which has access to vCenter or the vCenter itself.
+2. Restart the collector using:
+  ```sh
+  Restart-Service -Name OtelcolSumo
+  ```
+
+</TabItem>
+<TabItem value="macOS">
+
+1. Copy the yaml file to /etc/otelcol-sumo/conf.d/ folder in the VMWare instance which has access to vCenter or the vCenter itself.
+2. Restart the otelcol-sumo process using the below command 
+  ```sh
+  otelcol-sumo --config /etc/otelcol-sumo/sumologic.yaml --config "glob:/etc/otelcol-sumo/conf.d/*.yaml" 
+  ```
+
+</TabItem>
+<TabItem value="Chef">
+
+import ChefNoEnv from '../../../reuse/apps/opentelemetry/chef-without-env.md';
+
+<ChefNoEnv/>
+
+</TabItem>
+
+<TabItem value="Ansible">
+
+import AnsibleNoEnv from '../../../reuse/apps/opentelemetry/ansible-without-env.md';
+
+<AnsibleNoEnv/>
+
+</TabItem>
+
+<TabItem value="Puppet">
+
+import PuppetNoEnv from '../../../reuse/apps/opentelemetry/puppet-without-env.md';
+
+<PuppetNoEnv/>
+
+</TabItem>
+</Tabs>
+
+import LogsOutro from '../../../reuse/apps/opentelemetry/send-logs-outro.md';
+
+<LogsOutro/>
+
+
+## Sample queries
+
+### Metrics query
+
+This sample Query is from the **VMWare - Overview** > **Top 25 ESXi Hosts - CPU Utilization** panel.
+
+```sql title="Metric query"
+sumo.datasource=vmware  metric=vcenter.host.cpu.utilization vcenter.datacenter.name=* vcenter.cluster.name=* vcenter.host.name=* | topk(25,avg)
+```
+
+## Sample metrics
+
+### Metric message
+
+```json title="Metric message"
+{
+  "queryId": "A",
+  "_source": "WIN-U8TUICHO2JM",
+  "_metricId": "-nHlUFTEN-e35PT8M0bW4Q",
+  "_sourceName": "vmware",
+  "host.group": "sumovmware",
+  "os.type": "windows",
+  "sumo.datasource": "vmware",
+  "_sourceCategory": "otel/vmware",
+  "deployment.environment": "sumovmware",
+  "_contentType": "OpenTelemetry",
+  "host.name": "WIN-U8TUICHO2JM",
+  "metric": "vcenter.host.cpu.utilization",
+  "_collectorId": "00005AF310C86B33",
+  "vcenter.datacenter.name": "Production",
+  "_sourceId": "0000000000000000",
+  "unit": "%",
+  "vcenter.cluster.name": "Prod Cluster",
+  "_sourceHost": "WIN-U8TUICHO2JM",
+  "_collector": "WIN-U8TUICHO2JM",
+  "vcenter.host.name": "esx1.esxlab.com",
+  "max": 45.76,
+  "min": 0,
+  "avg": 5.65,
+  "sum": 1621.41,
+  "latest": 0,
+  "count": 287
+}
+```
+
+## Viewing VMWare dashboards
+
+### VMWare - Overview
```

Comment:
You should add a short description for each dashboard. See the [Kafka](http://localhost:3000/docs/integrations/containers-orchestration/opentelemetry/kafka-opentelemetry/#viewing-kafka-dashboards) and [ActiveMQ](http://localhost:3000/docs/integrations/containers-orchestration/opentelemetry/activemq-opentelemetry/#viewing-the-activemq-dashboards) articles for examples.

Reply:
Done